File: Pertemuan 5/Condition.java
import java.util.Scanner;

public class Condition {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // if
        System.out.print("Masukkan bilangan bulat = ");
        int x = scanner.nextInt();
        int sisa = x % 2;

        if(sisa == 0) {
            System.out.println("Anda memasukkan bilangan genap");
        } else {
            System.out.println("Anda memasukkan bilangan ganjil");
        }
        System.out.println("Done");

        // else if
        System.out.print("Masukkan umur Anda = ");
        int umur = scanner.nextInt();
        
        if(umur > 0 && umur <= 5) {
            System.out.println("Anda masih balita");
        } else if( umur > 5 && umur <= 19) {
            System.out.println("Anda sudah remaja");
        } else if(umur > 19 && umur <= 50) {
            System.out.println("Anda sudah dewasa");
        } else if( umur > 50) {
            System.out.println("Anda sudah tua");
        } else {
            System.out.println("Data tidak valid");
        }
        System.out.println("Done");
    }
}

File: Pertemuan 5/condition.py
# if
x = input("Masukkan bilangan bulat = ")
hasil = int(x) % 2

if (hasil == 0) :
    print("Anda memasukkan bilangan genap")
else :
    print("Anda memasukkan bilangan ganjil")
print("Done")

# else if / elif
umur = input("Masukkan umur Anda = ")
umur = int(umur)

if (umur > 0 and umur <=5) :
    print("Anda masih balita")
elif (umur > 5 and umur <= 19) :
    print("Anda sudah remaja")
elif (umur > 19 and umur <= 50) :
    print("Anda sudah dewasa")
elif (umur > 50) :
    print("Anda sudah tua")
else :
    print("Data tidak valid")
print("Done")